src/app/exercise-block/[exercise_block_id]/_components/WendlerBlockRowActive.tsx

```diff
@@ -97,7 +93,7 @@ const useWendlerBlockRowActiveAPI = (props: WendlerBlockRowActiveProps) => {
     onCompletionStatusChange,
     onNotesChange,
     onEffortChange: onPerceivedEffortChange,
-    onTargetWeightChange,
+    onTargetWeightChange: setTargetWeight,
```

Comment:
This is related to the bug I mentioned in the description. We had to switch this from using a callback that only supported a number, to the react setState format that allows you to use a closure. (honestly, this whole thing is _way_ overly complicated anyway, but that's out of scope to fix for now)